### PR TITLE
Bridge evm implementation initialization

### PR DESCRIPTION
**Pull request by Bridgerz**
## Description 

Addressing: https://dashboard.hackenproof.com/manager/sui/sui-protocol/reports/SUIPR-150 

Update the CommitteeUpgradeable contract to disable all upgradeable bridge contracts' implementations from initialising.

## Test plan 

Added a unit test.

### CODE REVIEW

File: bridge/evm/test/BridgeCommitteeTest.t.sol [hunk 1]
```diff
@@ -38,15 +38,19 @@ contract BridgeCommitteeTest is BridgeBaseTest {
...
 
     function testBridgeCommitteeInitializationLength() public {
-        BridgeCommittee _committee = new BridgeCommittee();
         address[] memory _committeeMembers = new address[](256);
 
         for (uint160 i = 0; i < 256; i++) {
             _committeeMembers[i] = address(i);
         }
 
+        // vm.expectRevert(bytes("BridgeCommittee: Committee length must be less than 256"));
```
Comment (longbowlu) [inline bridge/evm/test/BridgeCommitteeTest.t.sol:47]: remove

File: bridge/evm/test/CommitteeUpgradeableTest.t.sol [hunk 2]
```diff
@@ -204,6 +198,30 @@ contract CommitteeUpgradeableTest is BridgeBaseTest {
...
+        _stake[0] = 1000;
+        _stake[1] = 1000;
+        _stake[2] = 1000;
+        _stake[3] = 2002;
+        _stake[4] = 4998;
+
+        // Try to call the initializer directly on the implementation contract
+        // Expect it to revert because initializers should be disabled
+        vm.expectRevert();
+        BridgeCommittee(implementation).initialize(_committeeMembers, _stake, 100);
```
Comment (longbowlu) [inline bridge/evm/test/CommitteeUpgradeableTest.t.sol:222]: do we want to assert the same for every CommitteeUpgradable impl contract?

File: bridge/evm/script/deploy_bridge.s.sol [hunk 3]
```diff
@@ -123,6 +123,9 @@ contract DeployBridge is Script {
 
         // initialize config in the bridge committee
         BridgeCommittee(bridgeCommittee).initializeConfig(address(bridgeConfig));
+        BridgeCommittee committeeImplementation =
+            BridgeCommittee(Upgrades.getImplementationAddress(bridgeCommittee));
+        committeeImplementation.initializeConfig(address(bridgeConfig));
```
[the remark this answers is not in the document]
Comment (longbowlu) [reply bridge/evm/script/deploy_bridge.s.sol:128]: wait shouldn't this revert?

[the remark this answers is not in the document]
Comment (Bridgerz) [reply bridge/evm/script/deploy_bridge.s.sol:128]: this is "initializeConfig" which is a special case.

File: bridge/evm/test/CommitteeUpgradeableTest.t.sol (hunk 2 above)
Comment (Bridgerz) [reply bridge/evm/test/CommitteeUpgradeableTest.t.sol:222]: they all reference the same `_disableInitializers()` call in the base `CommitteeUpgradeable` contract so if it works for one it should work for all.

File: bridge/evm/script/deploy_bridge.s.sol (hunk 3 above)
[the remark this answers is not in the document]
Comment (longbowlu) [reply bridge/evm/script/deploy_bridge.s.sol:128]: ohhh

**Review by longbowlu — APPROVED** [review]
thanks!

**Pull request merged**